test/extended/fixtures/forcepull-false-s2i.json

```diff
@@ -21,7 +21,7 @@
             "sourceStrategy": {
 		"from": {
 		    "kind": "DockerImage",
-		    "name": "docker.io/openshift/ruby-20-centos7"
+		    "name": "s2i-forcepull-false-ruby-20-centos7"
```

Comment:
this probably falls into the realm of personal preference, but in all these places where you need to do a substitution before the json is usable, i'd do something like this instead:
"name": "SERVICE_REGISTRY_IP/s2i-forcepull-false-ruby-20-centos7" and then substitute on SERVICE_REGISTRY_IP.

1) you can substitute on the same string in every file
2) it's clear a substitution has to happen before the file is usable (for someone looking at the json)